src/tablib/formats/_ods.py

```diff
@@ -49,6 +52,93 @@ def export_book(cls, databook):
         wb.save(stream)
         return stream.getvalue()
 
+    @classmethod
+    def import_sheet(cls, dset, sheet, headers=True, skip_lines=0):
+        """Populate dataset `dset` with sheet data."""
+
+        dset.title = sheet.getAttribute('name')
+
+        def is_real_cell(cell):
+            return cell.hasChildNodes() or not cell.getAttribute('numbercolumnsrepeated')
+
+        for i, row in enumerate(sheet.childNodes):
+            if row.tagName != 'table:table-row':
+                continue
+            if i < skip_lines:
+                continue
+            row_vals = [cls.read_cell(cell) for cell in row.childNodes if is_real_cell(cell)]
+            if not row_vals:
+                continue
+            if i == skip_lines and headers:
+                dset.headers = row_vals
+            else:
+                if i > skip_lines and len(row_vals) < dset.width:
+                    row_vals += [''] * (dset.width - len(row_vals))
+                dset.append(row_vals)
+
+    @classmethod
+    def read_cell(cls, cell, value_type=None):
+        def convert_date(val):
+            if 'T' in val:
+                return datetime.strptime(val, "%Y-%m-%dT%H:%M:%S")
```

Comment:
And then refer to these as:
```suggestion
                return dt.datetime.strptime(val, "%Y-%m-%dT%H:%M:%S")
```
etc?